src/utils.js

```diff
@@ -34,3 +34,23 @@ export function useInterval({ callback, delay, runOnStart }) {
     }
   }, [delay, runOnStart]);
 }
+
+export function isFeatureEnabled(feature) {
+  const experimentalFeatures = localStorage.getItem('dopplerExpermiental')
+    ? JSON.parse(localStorage.getItem('dopplerExpermiental'))
+    : null;
+  if (experimentalFeatures) {
+    return !!experimentalFeatures.features.find((item) => item === feature);
+  }
+  return false;
+}
+
+export function getFeatureValue(param) {
+  const experimentalFeatures = localStorage.getItem('dopplerExpermiental')
+    ? JSON.parse(localStorage.getItem('dopplerExpermiental'))
+    : null;
+  if (experimentalFeatures) {
```

Comment:
Why not simply:

```js
return experimentalFeatures && experimentalFeatures.data[param];
```

Or

```js
return experimentalFeatures && experimentalFeatures.data[param] || false;
```

Reply:
Here I return a value or null, not true/false.